src/components/LocalFileUpload/custom-request-upload.ts

```diff
@@ -1,39 +1,113 @@
-import { RcCustomRequestOptions } from "antd/lib/upload/interface";
+import { SimulariumFileFormat } from "@aics/simularium-viewer";
+import { findIndex } from "lodash";
+import {
+    UploadRequestOption,
+    UploadRequestError,
+} from "rc-upload/lib/interface";
```

Comment:
The `UploadRequestOption` here is the same thing as `{ RcCustomRequestOptions } from "antd/lib/upload/interface"`, but with the Antd update the old import statement was not working, so I had to modify it